src/test/java/homework16.java

```diff
@@ -0,0 +1,33 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class BaseTest {
+
+    @Test
+    public void registrationNavigation() {
+        // Set the path to the ChromeDriver executable
+        System.setProperty("webdriver.chrome.driver", "path/to/chromedriver");
+
```

Comment:
Add this line of code:

`ChromeOptions options = new ChromeOptions();`
`options.addArguments("--remote-allow-origins=*");`